module/zfs/arc.c

```diff
@@ -7138,6 +7138,7 @@ arc_tempreserve_space(uint64_t reserve, uint64_t txg)
 
 	if (reserve + arc_tempreserve + anon_size > arc_c / 2 &&
 	    anon_size > arc_c / 4) {
+#ifdef ZFS_DEBUG
```

Comment:
Just to verify these additional `ZFS_DEBUG` blocks are to prevent the unused variable warnings correct?

Reply:
exactly